static/app/views/dashboards/manage/dashboardList.tsx

```diff
@@ -46,8 +50,22 @@ function DashboardList({
   dashboards,
   pageLinks,
   onDashboardsChange,
+  rows,
+  columns,
+  resizing,
 }: Props) {
   const navigate = useNavigate();
+
+  const [currentDashboards, setCurrentDashboards] = useState<
+    DashboardListItem[] | undefined
+  >(dashboards);
+
+  useEffect(() => {
+    if (dashboards?.length) {
+      setCurrentDashboards(dashboards);
+    }
+  }, [dashboards]);
+
```

Comment:
Is this acting as a cache? If yes, since `ManageDashboards` already uses `useApiQuery` I'm wondering if we can rely on _that_ cache instead.

Reply:
the problem was that the resize causes `useApiQuery` to be called again meaning dashboards is `[]` for a split second so the placeholders don't show

Reply:
I tried _so_ hard to find a tidy solution for this, but came up with nothing better than this. I think what you're looking for is the `usePrevious` hook! Give that a go

Aside, apparently it's possible to pull stuff directly out of TanStack cache? Crimes!

```jsx
const queryClient = useQueryClient();
const cache = queryClient.getQueryCache();

const lastKnownDashboards = cache.findAll({queryKey: ['/organizations/sentry/dashboards/']})[0].state.data;
```